src/common/collections.rs

```diff
@@ -54,7 +55,8 @@ pub async fn do_get_collection(
     name: &str,
     shard_selection: Option<ShardId>,
 ) -> Result<CollectionInfo, StorageError> {
-    let collection_pass = access.check_partial_collection_rights(name)?;
+    let collection_pass =
+        access.check_collection_access(name, false, CollectionAccessMode::Read)?;
```

Comment:
I think it should require access to the whole collection to execute this request

Reply:
Otherwise total number of points can leak